proprietary/tokens/src/common/amsterdam/spacing.tokens.json

```diff
@@ -6,6 +6,15 @@
         "md": { "value": "1rem" },
         "lg": { "value": "1.5rem" },
         "xl": { "value": "2.5rem" }
+      },
+      "outside": {
```

Comment:
Do we need to define these in `common` as well? Doesn't add a lot, imo, and it adds a bit of complexity.

Reply:
I’ve removed the in-between definition again, at least for now.